templates/confidential-computing/index.html

```diff
@@ -0,0 +1,287 @@
+                alt="AWS logo",
+                height="22",
+                width="40",
+                hi_def=True,
+                loading="lazy" ) | safe
+              }}
+            </p>
+          </div>
+          <div class="col-4 col-medium-3 col-small-2">
+            <p><a href="https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/snp-requirements.html">AMD SEV SNP with Ubuntu 23.10 on AWS</a></p>
+          </div>
+        </div>
+      </div>
+    </div>
+  </section>
+
+  <hr class="is-fixed-width" />
+
+  <section class="p-strip is-deep">
+    <div class="row">
+      <div class="col-12">
+        <p class="p-heading--2">
+          <a href="https://ubuntu.com/engage/introduction-confidential-computing">Download our guide to confidential computing ›</a>
+        </p>
+      </div>
+    </div>
+  </section>
+
+  <hr class="is-fixed-width" />
+
+  <section class="p-strip u-no-padding--top">
+    <div class="row--50-50">
+      <div class="col">
+        <h2>A solid foundation of your zero trust strategy</h2>
+      </div>
+      <div class="col">
+        <p>
+          With confidential computing, you can remove the privileged systems
+          software from your trusted computing base, reduce your attack surface,
+          and get a remotely verifiable cryptographic guarantee before trusting
+          any platform with your data.
+        </p>
+        <p>
+          <a href="https://ubuntu.com/blog/build-foundation-zero-trust-strategy-ubuntu-confidential-computing">Learn
+            more about zero trust and confidential computing ›</a>
+        </p>
+      </div>
+    </div>
+  </section>
+
+  <hr class="is-fixed-width" />
+
+  <section class="p-strip u-no-padding--top">
+    <div class="row--50-50">
+      <div class="col">
+        <h2>Confidential computing in your private data centre</h2>
+      </div>
+      <div class="col">
+        <p>
+          Combine good data governance practices with the latest advances in
+          privacy-enhancing computation. Use confidential computing to protect
+          the confidentiality and integrity of the sensitive data hosted on your
+          on-premises servers, using hardware-rooted primitives beyond
+          traditional measures.
+        </p>
+        <p>
+          <a href="https://canonical.com/blog/why-do-you-also-need-confidential-computing-for-your-private-datacenter">Why
+            use confidential VMs in your private datacenter? ›</a>
+        </p>
+      </div>
+    </div>
+  </section>
+
+  <hr class="is-fixed-width" />
+
+  <section class="p-strip is-deep u-no-padding--top">
+    <div class="row--50-50">
+      <div class="col">
+        <h2>Learn more about confidential computing</h2>
+      </div>
+      <div class="col">
+        <ul class="p-list--divided u-no-margin--bottom">
+          <li class="p-list__item">
+            <a href="https://ubuntu.com/blog/what-is-confidential-computing-a-high-level-explanation-for-cisos">What is
```

Comment:
```suggestion
            <a href="/blog/what-is-confidential-computing-a-high-level-explanation-for-cisos">What is
```

Reply:
good point, I was just copying the links directly from the copy doc 🤦‍♂️